packages/kbn-optimizer/limits.yml

```diff
@@ -67,12 +67,12 @@ pageLoadAssetSize:
   savedObjectsTagging: 59482
   savedObjectsTaggingOss: 20590
   searchprofiler: 67080
-  security: 189428
+  security: 95864
   securityOss: 30806
   securitySolution: 187863
   share: 99061
   snapshotRestore: 79032
-  spaces: 387915
+  spaces: 57868
```

Comment:
Not strictly required for this PR, but we did the work to drop the spaces bundle already, so we should make sure it stays that way